app/views/likes/_button.html.erb

```diff
@@ -1,7 +1,16 @@
+<style>
+  .lk{
+    border:none;
+    background-color: white;
+  }
+</style>
 <% like = current_user.likes.find_by(likeable: likeable) %>
 <% if like.nil? %>  
-  <%=button_to("Like", likes_path(likeable),  class:"btn btn-success",  params:{ like: {likeable_id: likeable.id,  likeable_type: likeable.class.to_s}})%>
+  <%=button_to likes_path(likeable), class:"lk", params:{ like: {likeable_id: likeable.id,  likeable_type: likeable.class.to_s}} do %>
+    <i class="fa-sharp fa-regular fa-heart"></i>
+  <%end%>
 <% else %>  
-  <%=button_to("Unlinke", like_path(like), class:"btn btn-success", method: :delete)%>
-  
+  <%=button_to like_path(like), class:"lk", method: :delete do%>
+    <i class="fa-sharp fa-solid fa-heart" style="color: #f70814;"></i>
+  <% end %>
```

Comment:
use fa-heart class in css file to add color

Reply:
okk